content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,208 @@
+{{% /collapse-content %}} 
+
+
+### Multilayer space aggregation
+
+Once you've already specified tag(s) in your first layer of space aggregation to group by, multilayer space aggregation can be accessed with the `Group By` function. 
+
+You can apply additional layers of space aggregation with subsequent `Group Bys`.
+Note: if you do not specify tag(s) to group by in your initial space aggregation layer, multilayer space aggregation will not be available.
+
+The first layer of space aggregation supports the following aggregators:
+
+- `avg by`
+- `sum by`
+- `min by`
+- `max by`
+
+Additional layers of space aggregation support:
+
+- `avg by`
+- `sum by`
+- `min by`
+- `max by`
+- `arbitrary percentile pXX` (`p75, p99, p99.99, etc.`)
+- `stddev by`
+
+Multilayer space aggregation can be used with the following functions: 
+| Supported Functions   | Description                                                                                    |
+|-----------------------|-----------------------------------------------------------------------------------------------|
+| Arithmetic operators   | `+, -, *, /`                                                                                  |
+| Timeshift functions    | `<METRIC_NAME>{*}, -<TIME_IN_SECOND>`<br> `hour_before(<METRIC_NAME>{*})`<br> `day_before(<METRIC_NAME>{*})`<br> `week_before(<METRIC_NAME>{*})`<br> `month_before(<METRIC_NAME>{*})` |
+| Top-k selection        | `top(<METRIC_NAME>{*}, <LIMIT_TO>, '<BY>', '<DIR>')`                                         |
+
+Any other functions not listed above cannot be combined with multilayer space aggregation. 
+
+All space aggregators with the exception of percentile space aggregators have one argument, which is the tag key(s) you'd want to group by. Percentile space aggregators require two arguments: 
+- The arbitrary percentile pXX
+- The tag(s) to group by
+
+
+{{% collapse-content title="Space aggregation example queries" level="h5" %}}
+This initial query, `avg:aws.ec2.cpuutilization{*} by {env,host}.rollup(avg, 300)` calculates the sum of average CPU utilization, grouped by `env` and `host` every 5 minutes. Then multilayer space aggregation is applied to calculate maximum value of the average CPU utilization by `env`.
+
+
+In the UI or JSON tab, it would look as follows:
+
+{{< img src="/metrics/nested_queries/multilayer-space-agg-ui.png" alt="example of multilayer space aggregation in the UI" style="width:100%;" >}}
+
+{{< img src="/metrics/nested_queries/multilayer-space-agg-json.png" alt="example of multilayer space aggregation in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+## Percentiles and Standard Deviation for Aggregated Counts/Rates/Gauges
+
+You can use multilayer aggregation (time / space) to query percentiles and standard deviation from queries on counts/rates/gauges. They allow you to better understand the variability and spread of your large datasets and allow you to better identify outliers. 
```

Comment:
```suggestion
You can use multilayer aggregation (time and space) to query percentiles and standard deviation from queries on counts, rates, and gauges. They allow you to better understand the variability and spread of your large datasets and allow you to better identify outliers. 
```

Reply:
What's the `They` in  "They allow you to better understand the variability and spread of your large datasets and allow you to better identify outliers."